### PR TITLE
Add New UI

### PR DESCRIPTION
**Pull request by ufukhawk**
*Thanks for submitting a PR to the Snppts ✂️ library!*

## Submission description

68747470733a2f2f696d6775722e636f6d2f6b47687a3171392e706e67

### Possible platform improvements

_Missing something on the platform? Let us know here._

### CODE REVIEW

File: src/Snppts/Snippets/ChasedHomeUIDesign.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        public GitHubRepoInfo GitHubRepoInfo => new GitHubRepoInfo("ufukhawk/Chased-Home-UI-Design");
+        public Uri ExternalUri => null;
+        public bool ContainsAndroidSample => true;
+        public bool ContainsiOSSample => true;
+        public bool ContainsUWPSample => false;
+        public IAmAnAuthor AuthorInfo => new UfukHawk();
+
+        public IEnumerable<Uri> ImageUris => new List<Uri>
+        {
+            new Uri("https://imgur.com/W4LqlFI.png"),
```
Comment (jfversluis) [inline src/Snppts/Snippets/ChasedHomeUIDesign.cs]: Please link to images, not websites. Preferably on github itself

**Review by jfversluis — CHANGES_REQUESTED** [review]

**Review by ufukhawk — COMMENTED** [review]
Edited Img Uri

**Review by ufukhawk — COMMENTED** [review]
Edit Namespace

**Review by ufukhawk — COMMENTED** [review]
Merge branch 'master' into patch-3

**Review by jfversluis — APPROVED** [review]

**Pull request merged**